lib/credentials/credential_provider_chain.d.ts

```diff
@@ -10,6 +10,10 @@ export class CredentialProviderChain extends Credentials {
      */
     resolve(callback:(err: AWSError, credentials: Credentials) => void): CredentialProviderChain;
     /**
+     * Return a Promise on resolve() function
+     */
+    resolvePromise(resolve?:(credentials: Credentials) => void, reject?:(err: AWSError) => void): Promise<any>;
```

Comment:
This isn't quite right. The return value should use `Credentials` instead of `any`, and the method itself doesn't take any callbacks.